### DISCUSSION TITLE
Relation not loaded

### DISCUSSION THREAD

**Discussion by gjm** [rappasoft/laravel-livewire-tables#871] [Q&A]
Let's say I have two models, `Employee` and `Company` and `Employee` belongsTo `Company`.

I want to have a simple datatable listing employees and respective companies. The column definition would be something like:

```php
public function columns(): array
{
    return [
        Column::make('Number')->searchable()->sortable(),
        Column::make('Name')->searchable()->sortable(),
        Column::make('Company', 'company.name')->searchable()->sortable(),
    ]
}
```

Relations are being eager loaded and everything works fine like this.

But, say I want to display the company's description, through the `getDescription()` method. If I chain `format(fn ($value, $row) => $row->getDescription())` to format the column's value I get an error saying I'm calling `getDescription()` on `null`.

I would expect, since the company name is being shown correctly and search and sorting also work fine, that the `company` relation of  each employee would be correctly loaded and set but it is not.

Am I doing something wrong?

**Comment by DanielGSoftware** [comment]
What does your query look like?

**Comment by gjm** [comment]
I reported an issue (#872) and there is more info there. 

Thanks for your answer.
